src/IndicatorsDash.jsx

```diff
@@ -56,7 +57,49 @@ class IndicatorsDash extends React.Component {
     };
   
     render() {
-      const { moduleName, connError, data } = this.state;
+      const { moduleName, connError, data, activeTab } = this.state;
+
+      const tabs = {
+        'all': 'Todas',
+        'riesgo': 'Riesgo Biodiversidad',
+        'costo': 'Costo de Compensación',
+        'oportunidad' :'Oportunidad',
+        'monitoreo' :'Monitoreo',
+        'bioma' :'Bioma'
+      };
+
+     console.log('activeTab: ' + activeTab);
```

Comment:
Please remove all console.log lines before pushing the code to the repository

Reply:
Done